app/models/work.rb

```diff
@@ -0,0 +1,14 @@
+class Work < ApplicationRecord
+  has_many :votes, dependent: :destroy
+  has_many :users, through: :votes
+  validates :title, presence: true, uniqueness: true
+  validates :category, presence: true, inclusion: { in: %w(album movie book) }
+  validates :publication_year, numericality: { only_integer: true }
+  scope :ordering, -> {order('votes_count DESC')}
+
+  CATEGORIES = %w[album book movie]
+
+  def self.get_categories
+    return CATEGORIES
+  end
```

Comment:
Nicely done!!! I really like this class method. Side note, however... Turns out that you can access constants defined in a class with the following syntax: `Work::CATEGORIES` (which would replace this class method `Work.get_categories`). You can do that in rails console to check!